Source/WebCore/dom/Document.cpp

```diff
@@ -6586,6 +6587,9 @@ std::optional<RenderingContext> Document::getCSSCanvasContext(const String& type
     if (is<ImageBitmapRenderingContext>(*context))
         return RenderingContext { RefPtr<ImageBitmapRenderingContext> { &downcast<ImageBitmapRenderingContext>(*context) } };
 
+    if (is<GPUCanvasContext>(*context))
```

Comment:
Good use case for dynamicDowncast

Reply:
The code around the area follows the same `is<...>(...)` pattern, so it'd stand out if we use `dynamicDowncast` here.

Reply:
Yes, it stands out as beautiful :)